src/txn/txn.c

```diff
@@ -572,11 +572,14 @@ __wt_txn_update_oldest(WT_SESSION_IMPL *session, uint32_t flags)
 
         /* Output a verbose message about long-running transactions,
          * but only when some progress is being made. */
+        current_id = __wt_atomic_loadv64(&txn_global->current);
+        WT_ASSERT(session, WT_TXNID_LE(oldest_id, current_id));
         if (WT_VERBOSE_ISSET(session, WT_VERB_TRANSACTION) &&
           current_id - oldest_id > (10 * WT_THOUSAND) && oldest_session != NULL) {
             __wt_verbose(session, WT_VERB_TRANSACTION,
-              "old snapshot %" PRIu64 " pinned in session %" PRIu32 " [%s] with snap_min %" PRIu64,
-              oldest_id, oldest_session->id, oldest_session->lastop,
+              "current snapshot %" PRIu64 ", old snapshot %" PRIu64 " pinned in session %" PRIu32
```

Comment:
The original message is poorly worded and unfortunately the new message is also poorly worded. 

The `oldest_id` that is logged is not a "snapshot" although would be related to the oldest open sessions snapshot, most likely it's snap_min. The "current snapshot" is really the next transaction id that will exist in the system, so that is mislabelling.

So I would update this to say:
"Oldest id ... pinned in session ... with snap_min ..." I don't think we should log the `current_id` here as this message is meant to be informational about the oldest session / long running transaction.